examples/scala_rockstar.py

```diff
@@ -0,0 +1,5 @@
+from RockStar import RockStar
+
+scala_code = "object HelloWorld {def main(args: Array[String]) { println(\"Hello world\") }}"
```

Comment:
Any reason you are delimiting string quotes... Following wouldn't work?

"object HelloWorld {def main(args: Array[String]) { println('Hello world') }}"

or scala expects `"`